src/components/icon/icon.jsx

```diff
@@ -11,6 +11,13 @@ export default function Icon(props) {
     props.currentColor === 'skip' ? '' : styles[`color-${props.currentColor}`],
   ].filter(Boolean);
 
+  const tabindex = props.tabable ? '0' : undefined;
```

Comment:
We should base this on `props.role`: if it is interactive then we set a tabindex of `0` otherwise `1`